src/linagora.esn.unifiedinbox/app/app.js

```diff
@@ -214,7 +202,7 @@ require ('./components/subheader/not-spam-button/subheader-not-spam-button.js');
 require ('./components/user-quota/user-quota.controller.js');
 require ('./components/user-quota/user-quota.js');
 require ('./config.js');
-require ('./constants.js');
+require ('../../esn.inbox.libs/app/app.constants.js');
```

Comment:
Now, `esn.inbox.libs` is an Angular module, so require the module file in app.js is enough, no need to require its services, constants, etc. Please remove something like this and in other files in `linagora.esn.unifiedinbox` module.